spec/inputs/jms.yml

```diff
@@ -0,0 +1,18 @@
+---
+# HornetQ Broker
+hornetq:
+  # Connect to a local HornetQ Broker using JNDI
+  :jndi_name: /ConnectionFactory
+  :jndi_context:
+    java.naming.factory.initial: org.jnp.interfaces.NamingContextFactory
+    java.naming.provider.url: jnp://localhost:1099
+    java.naming.factory.url.pkgs: org.jboss.naming:org.jnp.interfaces
+    java.naming.security.principal: guest
+    java.naming.security.credentials: guest
+  :require_jars:
```

Comment:
The problem with doing this is that it would make it harder to swap out JMS providers. The plugin does not target a specific JMS provider hornetq just happens to be the one I tested on. My understanding is if I wanted to do it this way it would likely be harder to target a different JMS provider. Is this not the case?